crates/sui-protocol-config/src/lib.rs

```diff
@@ -612,7 +612,7 @@ impl ProtocolConfig {
                 max_object_vec_argument_size: Some(128),
                 max_coins: Some(1024),
                 max_pay_recipients: Some(1024),
-                max_programmable_tx_commands: Some(128),
+                max_programmable_tx_commands: Some(1024),
```

Comment:
@oxade @sadhansood @asonnino I've had to bump this up for benchmarks which generate pay transactions with 500 recipients, which means ~1000 commands.

We could drop the chunk in the benchmark config (https://github.com/MystenLabs/sui/blob/7b60006a268e7cec1c45250b3b655af8418299ca/crates/sui-benchmark/src/options.rs#L45) or solve this some other way, I'm not sure.